source/docs/api-reference/wpilib-integration/motorcontroller-integration.rst

```diff
@@ -0,0 +1,79 @@
+
+Phoenix Pro implements all inherited functions:
+
+- ``set``
+- ``setVoltage``
+- ``get``
+- ``setInverted``
+- ``getInverted``
+- ``disable``
+- ``stopMotor``
+
+Due to inheriting the ``MotorController`` interface, this allows motors such as ``TalonFX`` to be included as part of the ``DifferentialDrive`` constructor.
+
+.. tab-set::
+
+   .. tab-item:: Java
+      :sync: Java
+
+      .. code-block:: java
+
+         // instantiate motor controllers
+         TalonFX m_motorLeft = new TalonFX(0);
+         TalonFX m_motorRight = new TalonFX(1);
+
+         // create differentialdrive object for robot control
+         DifferentialDrive m_diffDrive = new DifferentialDrive(m_motorLeft, m_motorRight);
+
+         // instantiate joystick
+         XboxController m_driverJoy = new XboxController(0);
+
+         public void teleopPeriodic() {
+            var forward = -m_driverJoy.getLeftY();
+            var rot = -m_driverJoy.getRightX();
+
+            m_diffDrive.arcadeDrive(forward, rot);
+         }
+
+   .. tab-item:: C++ (Source)
+      :sync: C++
+
+      .. code-block:: cpp
+
+         public void TeleopPeriodic() {
+            auto forward = -driverJoy.GetLeftY();
+            auto rot = -driverJoy.GetRightX();
+
+            diffDrive.ArcadeDrive(forward, rot);
+         }
+
+   .. tab-item:: C++ (Header)
+      :sync: C++ Header
+
+      .. code-block:: cpp
+
+         #include "frc/drive/DifferentialDrive.h"
+         #include "frc/XboxController.h"
+
+         // instantiate motor controllers
+         hardware::TalonFX motorLeft{0};
+         hardware::TalonFX motorRight{1};
+
+         // create differentialdrive object for robot control
+         frc::DifferentialDrive diffDrive{motorLeft, motorRight};
+
+         // instantiate joystick
+         frc::XboxController driverJoy{0};
+
+Simulation
+----------
+
+It's recommended that users set supply voltage to ``getBatteryVoltage()`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/RobotController.html#getBatteryVoltage()>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc_1_1_robot_controller.html#a4b1e42e825583c82664a4ecc5d81b83f>`__) to take advantage of WPILib's ``BatterySim`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/simulation/BatterySim.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc_1_1sim_1_1_battery_sim.html>`__). Additionally, all signal states are shown in the simulation :guilabel:`Other Devices` menu.
```

Comment:
I don't like the wording of "all signal states are shown", maybe "the simulated device state is shown"?